app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ActiveRecord::Base
+  belongs_to :user
```

Comment:
A posts belongs to an `author` as you use a `custom foreign key` named `author_id` in posts table.
`For example: belongs_to :author, class_name: 'User'`

Reply:
### Hello @SajjadAhmad14 , 
Thank you for reviewing my work but I don't think this change you are requesting is part of the requirement Microverse gave. Please review the requirements again. 
Thank you regardless.